packages/prisma/migrations/20230629124638_membership_autoinc_id/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - The primary key for the `Membership` table will be changed. If it partially fails, the table could be left without primary key constraint.
+  - A unique constraint covering the columns `[userId,teamId]` on the table `Membership` will be added. If there are existing duplicate values, this will fail.
+
```

Comment:
There will not be any duplicates due to the fact this was used as the PK before hand.